tests/test_evaluator.py

```diff
@@ -229,11 +239,59 @@ def requires_hydrogen(self):
         def smaller_is_better(self):
             return False
 
-    evaluator = peppr.Evaluator([BrokenMetric()], tolerate_exceptions=True)
+    evaluator = peppr.Evaluator(
+        [BrokenMetric()], match_method, tolerate_exceptions=True
+    )
     system_id = list_test_predictions()[0]
     reference, poses = assemble_predictions(system_id)
     with pytest.warns(
         peppr.EvaluationWarning,
         match=f"Failed to evaluate Broken on '{system_id}': Expected failure",
     ):
         evaluator.feed(system_id, reference, poses[0])
+
+
+@pytest.mark.parametrize(
+    ["model_num", "optimal_bisy_rmsd"],
+    [
+        (1, 5.82),
+        (2, 11.49),
+        # For this one model the exhaustive method actually finds the optimal result
+        # (3, 7.68),
```

Comment:
Perhaps I don't understand the test fully, but I would expect that this assertion still passes
```python
    assert individual_bisy_rmsd <= optimal_bisy_rmsd + TOLERANCE
```
Is it simply that for this case
```python
    assert exhaustive_bisy_rmsd > optimal_bisy_rmsd + TOLERANCE
```
does not hold true?

Reply:
Yes, exactly. This test specifically checks cases where `HEURISTIC` and `EXHAUSTIVE` or insufficient. In this single case they are sufficient.